### DISCUSSION TITLE
Game crash

### DISCUSSION THREAD

**Discussion by coccmocc** [ppy/osu#26827] [Q&A]
### Type

Crash to desktop

### Bug description

in the first 5 minutes of the game the game crashed, I just opened the skin editor and moved the combo counter

- System 

  - Provider 

   [ Name]  .NET Runtime 
 
  - EventID 1026 

   [ Qualifiers]  0 
 
   Version 0 
 
   Level 2 
 
   Task 0 
 
   Opcode 0 
 
   Keywords 0x80000000000000 
 
  - TimeCreated 

   [ SystemTime]  2024-01-30T17:46:52.3826071Z 
 
   EventRecordID 24261 
 
   Correlation 
 
  - Execution 

   [ ProcessID]  0 
   [ ThreadID]  0 
 
   Channel Application 
 
   Computer HOME-PC 
 
   Security 
 

- EventData 

   Application: osu!.exe CoreCLR Version: 6.0.2623.60508 .NET Version: 6.0.26 Description: The process was terminated due to an unhandled exception. Exception Info: exception code c0000005, exception address 00007FF9F8EAE789  

### Version

2024.130.2

### Logs

[compressed-logs.zip](https://github.com/ppy/osu/files/14102319/compressed-logs.zip)

**Comment by bdach** [comment]
Can you check if there are any driver updates for your graphics card?

**Reply by coccmocc** [reply]
yes, there is an update,but the regular osu also crashed f(this happens randomly, most often when I stream on Discord)

**Reply by coccmocc** [reply]
and,should i update my amd driver???

**Reply by bdach** [reply]
> but the regular osu also crashed f(this happens randomly, most often when I stream on Discord)

I'd say it's most likely that it's discord provoking those crashes then. Updating drivers may help in that case still.

**Reply by coccmocc** [reply]
ok, wait until I update these drivers (I've been struggling for a long time)

**Reply by coccmocc** [reply]
Eh...I decided to upgrade Windows additionally and now I have an endless "Windows preparation do not turn off the computer"

**Reply by coccmocc** [reply]
![image](https://github.com/ppy/osu/assets/158207382/3367f5c1-5043-4bac-b85d-bac35b73d623)

what is this???

**Reply by bdach** [reply]
> what is this???

that's kinda outdated now. it was there just so that if anything goes wrong during the pp rollout people would know that's sort of expected.

**Reply by coccmocc** [reply]
![image](https://github.com/ppy/osu/assets/158207382/979c5ec9-fa7c-4324-8f25-e28dec02aa31)

![image](https://github.com/ppy/osu/assets/158207382/c93012ce-5708-4923-bda3-b1229e1a2f49)



other problem :(

**Reply by bdach** [reply]
i'm not sure i understand what i'm looking at here, can you describe the issue in more detail?

**Reply by coccmocc** [reply]
I agree, the screenshots turned out poor... In short, in the editor the life bar is close to the notes, but in the game it’s not (I’m writing through a translator, I hope everything is clear...)

**Reply by coccmocc** [reply]
In the skin editor, some elements are shifted to the right (life bar and accuracy) + osu crashed again

**Reply by bdach** [reply]
> In the skin editor, some elements are shifted to the right (life bar and accuracy)

You can use the "playfield" skin layer to attach things to the mania playfield rather than try and align them in the default one.

**Reply by coccmocc** [reply]
thanks)

**Reply by coccmocc** [reply]
osu crashes so far only in the skin editor

**Reply by bdach** [reply]
Is it the same crash as in the post above? If not, can you post logs again?

**Reply by coccmocc** [reply]
[compressed-logs.zip](https://github.com/ppy/osu/files/14103173/compressed-logs.zip)

**Reply by bdach** [reply]
Can you please try switching renderer in settings to either `OpenGL (Legacy)` or `Direct3D 11` and seeing if that improves things?

**Reply by coccmocc** [reply]
Yes,on direct3d all perfect


but problem with skin editor or with my mind



![image](https://github.com/ppy/osu/assets/158207382/223a2ceb-74e6-430f-abc2-2dc91afc398c)
